app/services/listing_service/store/shape.rb

```diff
@@ -137,6 +151,7 @@ def from_model(shape_model)
       hash[:units] = shape_model.listing_units.map { |unit_model|
         to_unit(from_unit_model_attributes(EntityUtils.model_to_hash(unit_model)))
       }
+      hash[:category_ids] = shape_model.categories.map { |category| category.id }
```

Comment:
`.pluck(:id)`

So we do this now every time to avoid multiple presentations even if in most places we don't need the category IDs?